docs/reference/mapping/params/index-prefixes.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[index-prefix-config]]
```

Comment:
I think we need to include this file from somewhere?

Reply:
also the rest of the PR suggests you wanted to call it `index-prefixes` rather than `index-prefix-config`